index.html

```diff
@@ -24,6 +25,7 @@
       >
         Create Tasks
       </button>
+      <a class="action-button" href="goal/index.html"> Create Goals </a>
       <button
         class="profile-task-btn"
         type="button"
```

Comment:
This looks off to me. Some elements are buttons here, some are anchors. Either add them as the other elements were added and create a separate PR to convert them all to button/anchors  or convert them all in this one itself however @prakashchoudhary07 thinks it should be done

Reply:
changed